### PR TITLE
README.md rewritten in English

### PR DESCRIPTION
**Pull request by Y-Havryliv**

### CODE REVIEW

**Review by MasterpieceElbow — APPROVED** [review]

**Review by Abnormaltype — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -100,18 +99,18 @@ errors = {
     ],
 }
 ```
-Здесь `errors` это словарь, в котором ключи - путь к файлу, а значение этого ключа -
-список ошибок, где каждая ошибка - словарь
+Here `errors` is a dictionary, where the keys are the path to the file,
```
Comment (alenatovstukha) [inline README.md:102]: Here `errors` is a dictionary, where the keys are the path to the file, and the value of this key is a list of errors, where each is a dictionary.

File: README.md [hunk 2]
```diff
@@ -100,18 +99,18 @@ errors = {
...
+Here `errors` is a dictionary, where the keys are the path to the file, 
+and the value of this key is a list of errors, where each error is a dictionary.
 
-Но мы храним результаты выполнения немного в другом формате.
-Напиши 3 функции, чтобы преобразовать отчет в нужном нам формате.
-1. `format_linter_error` - форматирует отдельно взятую ошибку
-2. `format_single_linter_file` - форматирует все ошибки для конкретного файла
-3. `format_linter_report` - форматирует все ошибки для всех файлов в отчете
+But we store the execution results in a slightly different format.
+Write 3 functions to convert the report in the format we need.
```
Comment (alenatovstukha) [inline README.md:106]: Write 3 functions to convert the report into the format we need:

File: README.md [hunk 3]
```diff
@@ -100,18 +99,18 @@ errors = {
...
+1. `format_linter_error` - formats a single error.
```
Comment (alenatovstukha) [inline README.md:107]: - `format_linter_error` — formats a single error;
- `format_single_linter_file` — formats all errors for a particular file;
- `format_linter_report` — formats all errors for all report files.

File: README.md [hunk 4]
```diff
@@ -100,18 +99,18 @@ errors = {
...
+2. `format_single_linter_file` - formats all errors for a particular file.
+3. `format_linter_report` - formats all errors for all files in the report.
 
-Все функции должны вмещать в себе только конструкцию `return`
+All functions must contain only the `return` construct.
```
Comment (alenatovstukha) [inline README.md:111]: construct -> keyword?

File: README.md [hunk 5]
```diff
@@ -2,8 +2,7 @@
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
 
-Линтер flake8 выдает такой отчет об ошибках при проверке каталога с файлами исходного 
-кода:
+The flake8 linter gives the following error report when checking a directory with source code files:
```
Comment (alenatovstukha) [inline README.md:5]: flake8 кодом не нужно выделить?

**Review by alenatovstukha — APPROVED** [review]

**Review by alenatovstukha — APPROVED** [review]

**Pull request merged**